pkg/bootstrap/docker/openshift/ansible.go

```diff
@@ -0,0 +1,182 @@
+package openshift
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"text/template"
+
+	"github.com/golang/glog"
+
+	"github.com/openshift/origin/pkg/bootstrap/docker/errors"
+	"github.com/openshift/origin/pkg/bootstrap/docker/run"
+)
+
+const (
+	defaultAnsibleImageUser      = "root"
+	defaultOpenshiftAnsibleImage = "openshift-ansible"
```

Comment:
For now there doesn't look to be any version-specific images on docker hub. Will we have them eventually? Or should the version default to "latest" ?